src/drivers/differential_pressure/asp5033driver/parameters.c

```diff
@@ -0,0 +1,12 @@
+/**
+ * ASP5033 differential pressure sensor (external I2C)
+ *
+ * @reboot_required true
+ * @group Sensors
+ * @boolean
+ */
+PARAM_DEFINE_INT32(SENS_EN_ASP5033, 0);
```

Comment:
You need to add starting this driver in the startup script here:
https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d/rc.sensors